components/badge-on-avatar.css

```diff
@@ -0,0 +1,36 @@
+.online-badge {
+    display: inline-block;
+    position: relative;
+}
+
+.green-dot {
```

Comment:
make a dot class with all the basic styling and then make a green dot class for green color and for red repectively